tools_docker/Debian_Uwsgi/common.sh

```diff
@@ -16,6 +16,8 @@ if [ ! -f /etc/tracim/development.ini ]; then
     sed -i "s|^basic_setup.preview_cache_dir = .*|basic_setup.preview_cache_dir = /var/tracim/data/preview|g" /etc/tracim/development.ini
     sed -i "s|^basic_setup.sessions_data_root_dir = .*|basic_setup.sessions_data_root_dir = /var/tracim/data|g" /etc/tracim/development.ini
     sed -i "s|^basic_setup.api_key =.*|basic_setup.api_key = $KEY|g" /etc/tracim/development.ini
+    sed -i "s|^session.type =.*|session.type = ext:redis|g" /etc/tracim/development.ini
```

Comment:
```suggestion
    sed -i "s|^session.type = .*|session.type = ext:redis|g" /etc/tracim/development.ini
```
Without space, regex not working.

Reply:
in fact, regex does work without spacing (.* will match space too), but you're right it's better this way.